website_blog_group_old_archives/controllers/website_blog.py

```diff
@@ -70,7 +70,7 @@ def blog(self, blog=None, tag=None, page=1, **opt):
         self._date_begin, self._date_end = opt.get('date_begin'), opt.get('date_end')
         self._tag = tag
         result = super(website_blog, self).blog(
-                blog=blog, tag=tag, page=page, **opt)
+                blog=blog, tag=tag, page=page, **opt.get('opt'))
```

Comment:
see

https://github.com/OCA/OCB/blob/8.0/addons/website_blog/controllers/main.py#L106

opt is expected to be a dict with opt,  start_date, end date.....